app/models/manageiq/providers/hawkular/middleware_manager/middleware_diagnostic_report.rb

```diff
@@ -108,7 +120,9 @@ def jdr_report_succeded(data)
       end
 
       $mw_log.info("#{log_prefix} Generation of JDR report [#{id}] [#{binary_blob.name}] succeded.")
-      @finish_signal << STATUS_READY
+
+      # Unblock main thread at generate_diagnostic_report method
+      @finish_signal << 'https://youtu.be/-Elk_9fVo9E'
```

Comment:
What are these youtube links?

Reply:
Oh, sorry. I was in a moment doing dumb things. Fixed.

Reply:
Also, see https://github.com/ManageIQ/manageiq-providers-hawkular/pull/30#discussion_r135082037

Reply:
Is an easter :egg:

Reply:
It was cool, sadly is gone :(